test/tests/outputs/postprocessor/output_pps_hidden_shown_check.i

```diff
@@ -0,0 +1,90 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = 0
+  xmax = 1
+  ymin = 0
+  ymax = 1
+  nx = 10
+  ny = 10
+  elem_type = QUAD4
+  # This test uses ElementalVariableValue postprocessors on specific
+  # elements, if you use DistributedMesh the elements get renumbered.
+  parallel_type = replicated
+[]
+
+[Functions]
+  [./ffn]
+    type = ParsedFunction
+    value = -4
+  [../]
+
+  [./exactfn]
+    type = ParsedFunction
+    value = x*x+y*y
+  [../]
+[]
+
+[Variables]
+  [./u]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[Kernels]
+  [./td]
+    type = TimeDerivative
+    variable = u
+  [../]
+
+  [./diff]
+    type = Diffusion
+    variable = u
+  [../]
+
+  [./force]
+    type = UserForcingFunction
+    variable = u
+    function = ffn
+  [../]
+[]
+
+[BCs]
+  [./all_u]
+    type = FunctionDirichletBC
+    variable = u
+    boundary = '0 1 2 3'
+    function = exactfn
+  [../]
```

Comment:
Well, this was the setup for solving a problem with know solution, u=x^2+y^2. But, it looks it got bastardized by adding the time dependency. So, yes, I can solve something simpler instead...